### ISSUE TITLE
amplify env checkout doesn't finish. Windows 10 wsl 2.

### ISSUE THREAD

**Issue by damianrr** [aws-amplify/amplify-cli#4819]
Hello, 

I recently moved my PC from linux to WSL in Windows and I'm having a problem with switching backend environments. When I run  `amplify env checkout <myenvname>` it spins forever and never loads the environment.
I'm using WSL 2 which according to Microsoft is not a system call translation layer but a fully fledged Linux kernel.
Other than that the app works fine.

I'm using Amplify CLI version 4.24.0

Any experiences with this problem or any idea on how to debug it down?
Thanks.

**Comment by UnleashedMind** [comment]
@damianrr 
Could you take a screenshot and show us where it got stuck?

**Comment by damianrr** [comment]
![Capture](https://user-images.githubusercontent.com/638567/87351191-13837480-c527-11ea-8257-64239ab6ea53.PNG)

Now I realized that the behavior is a bit weirder (hopefully it helps debugging it down) here's a description:
- I've 2 backend envs `dev` and `prod`
- I realized that I can switch from prod to dev correctly
- Trying to switch from dev to prod give rise to this endless spinning that the image depicts (I leaved it all night)
- If after a few seconds I break the process manually and do a `amplify status` I'm seeing that it indeed switched to `prod` env.

So basically is not that it doesn't work but more like it keeps spinning even after it changed the environment instead of releasing control to the shell again.

**Comment by jhockett** [comment]
Possible duplicate of #4795

**Comment by SwaySway** [comment]
@damianrr 
What categories do you have enabled in your project?

**Comment by damianrr** [comment]
I've Auth, API, Function and Analytics. I did some tests and can confirm that this issue has nothing to do with WSL or windows but with the amplify-cli version, namely: 4.24.x. 
I regressed to 4.23.0 and it works properly.

**Comment by jhockett** [comment]
Thanks for confirming @damianrr. Closing this as a duplicate of #4795
